homeassistant/scripts/benchmark/__init__.py

```diff
@@ -0,0 +1,74 @@
+"""Script to run benchmarks."""
+import asyncio
+import argparse
+from contextlib import suppress
+import logging
+import time
```

Comment:
Why not timeit? https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/helpers/entity.py#L5